openmetadata-service/src/main/java/org/openmetadata/service/resources/dqtests/TestCaseResource.java

```diff
@@ -1000,6 +1000,31 @@ public TableData getFailedRowsData(
     return repository.getSampleData(testCase, authorizePII);
   }
 
+  @DELETE
+  @Path("/{id}/failedRowsSample")
+  @Operation(
+      operationId = "deleteFailedRowsSample",
+      summary = "Delete failed rows sample data",
+      description = "Delete a sample of failed rows for this test case.",
+      responses = {
+        @ApiResponse(responseCode = "200", description = "OK"),
+        @ApiResponse(
+            responseCode = "404",
+            description = "Failed rows sample data for test case {id} is not found.")
+      })
+  public Response deleteFailedRowsData(
+      @Context UriInfo uriInfo,
+      @Context SecurityContext securityContext,
+      @Parameter(description = "Id of the table", schema = @Schema(type = "UUID")) @PathParam("id")
+          UUID id) {
+    OperationContext operationContext =
+        new OperationContext(entityType, MetadataOperation.DELETE_SAMPLE_DATA);
```

Comment:
@sushi30 my suggestion here is to define a additional MetadataOperation, it can be DELETE_FAILED_ROWS_OF_A_TEST_CASE

Reply:
I used `DELETE_TEST_CASE_FAILED_ROWS_SAMPLE` so its in form of:
`<OP>_<RESOURCE>_<SUB_RESOURCE>`